src/plugins/timeConductor/Conductor.vue

```diff
@@ -88,15 +88,24 @@
                 </div>
 
                 <conductor-axis
-                        class="c-conductor__ticks"
-                        :bounds="rawBounds"
-                        @panAxis="setViewFromBounds"></conductor-axis>
+                    class="c-conductor__ticks"
+                    :bounds="rawBounds"
+                    :isFixed="isFixed"
+                    @panAxis="setViewFromBounds"
+                    @zoomAxis="setViewFromBounds"
+                ></conductor-axis>
 
             </div>
             <div class="c-conductor__controls">
-                <!-- Mode, time system menu buttons and duration slider -->
                 <ConductorMode class="c-conductor__mode-select"></ConductorMode>
                 <ConductorTimeSystem class="c-conductor__time-system-select"></ConductorTimeSystem>
+                <ConductorHistory
+                    v-if="isFixed"
+                    class="c-conductor__history-select"
+                    :bounds="rawBounds"
```

Comment:
We probably actually don't want the "raw" bounds being passed down here (not great naming on my part, admittedly). The raw bounds are updated rapidly when the axis is panned. They are handled separately to the canonical bounds tracked by openmct.time.bounds() because we want to update the start and end bounds displayed in the time conductor, but we don't want to update all of our plots, tables, etc. until the user releases the mouse button. As such, we probably only want to update our bounds history on bounds changes that views actually react to. That is, when `openmct.time.on('bounds', ...);` is triggered.